src/diffusers/pipelines/stable_diffusion/README.md

```diff
@@ -114,13 +114,12 @@ from io import BytesIO
 
 from diffusers import CycleDiffusionPipeline, DDIMScheduler
 
-
-# load the scheduler. CycleDiffusion only supports stochastic schedulers.
-
 # load the pipeline
 # make sure you're logged in with `huggingface-cli login`
 model_id_or_path = "CompVis/stable-diffusion-v1-4"
-scheduler = DDIMScheduler.from_config(model_id_or_path, subfolder="scheduler")
```

Comment:
I think the following two lines are more or less identical. Could you compare the outputs of the two - they are exactly the same for me:

```py
from diffusers import DDIMScheduler

model_id_or_path = "CompVis/stable-diffusion-v1-4"
scheduler = DDIMScheduler.from_config(model_id_or_path, subfolder="scheduler", steps_offset=0)

scheduler2 = DDIMScheduler(beta_start=0.00085, beta_end=0.012, beta_schedule="scaled_linear",
                          num_train_timesteps=1000, clip_sample=False, set_alpha_to_one=False)

assert scheduler.config == scheduler2.config
```

Note that the `from_pretrained(...)` method is preferred as explained here: https://huggingface.co/docs/diffusers/main/en/using-diffusers/loading#loading-customized-pipelines

There is only one difference and that's the `steps_offset` - however I think we should follow Stable Diffusion here and set it to `1` which means we should leave the code just as it is

Reply:
Yes ``scheduler``, ``scheduler2``, and ``DDIMScheduler.from_config(model_id_or_path, subfolder="scheduler")`` all produce the same output. I think we should leave it as is. Thanks for your clarification!